lib/actions/promote.coffee

```diff
@@ -0,0 +1,65 @@
+###
+Copyright 2016 Resin.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	 http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+###
+
+module.exports =
+	signature: 'promote [deviceIp]'
+	description: 'Promote a resinOS device'
+	help: '''
+		If you're running Windows, this command only supports `cmd.exe`.
+
+		Use this command to promote your device.
+
+		Examples:
+
+			$ rdt promote
+			$ rdt promote --port 22222
+	'''
+	primary: true
+	options: []
+	action: (params, options, done) ->
+		child_process = require('child_process')
+		Promise = require 'bluebird'
+		_ = require('lodash')
```

Comment:
Why are you requiring locally rather than file scope? If that's an `rdt` thing then fair enough but it seems like it's better to define it at file scope.

I'd also add an empty line between internal and external modules, just helps to show what's going on.